src/main/java/net/dv8tion/jda/api/entities/MessageEmbed.java

```diff
@@ -93,17 +93,6 @@
      */
     public static final int EMBED_MAX_LENGTH_BOT = 6000;
 
-    /**
-     * The maximum amount of total visible characters an embed can have
-     * <br>This limit depends on the current {@link net.dv8tion.jda.api.AccountType AccountType} and applies to CLIENT
-     *
-     * @see net.dv8tion.jda.api.EmbedBuilder#setDescription(CharSequence)
-     * @see net.dv8tion.jda.api.EmbedBuilder#setTitle(String)
-     * @see net.dv8tion.jda.api.EmbedBuilder#setFooter(String, String)
-     * @see net.dv8tion.jda.api.EmbedBuilder#addField(String, String, boolean)
-     */
-    public static final int EMBED_MAX_LENGTH_CLIENT = 2000;
```

Comment:
I don't see why we would remove this? It can still be used to know the limits for a client account.

Reply:
But isn't JDA the wrong place for that since it completely dropped Client support? Not that it would be in interest of any JDA developer.

I looked at the Discord Documentation and I can't find any mention about that, perhaps the limit was dropped too?

Reply:
Doesn't matter anyway since this would be a breaking change.

Reply:
Yeah, true. Then I would add a deprecation notice that it will be removed in 4.4.0 or so. Okay?

Imo it should still be completely removed at some point.

Reply:
This field could still be useful for configuring max string lengths in a database or something, I don't really think it needs to be removed or deprecated.

Reply:
The value of `MessageEmbed.EMBED_MAX_LENGTH_CLIENT` is identical to ([`Message.MAX_CONTENT_LENGTH`](https://github.com/DV8FromTheWorld/JDA/blob/8bfe22fd24f012c47bf9e1e8111ee878319b435d/src/main/java/net/dv8tion/jda/api/entities/Message.java#L141-L147)). So this constant can be used instead if it is really needed. Users can also only use Embeds if they use a selfbot or something similar so it makes even less sense for this constant to exist imo.

Reply:
I myself use this in different places where I make pagination embeds to check when I should split and end in a new embed, so it can have its uses.